config/root-config.in

```diff
@@ -333,9 +333,9 @@ win32)
       auxcflags="$auxcflags -GR -GX -G5"
    fi
    auxlibs=
-   forcelibs=$rootulibs
-   forceglibs=$rootuglibs
-   forceevelibs=$rootuevelibs
+   #forcelibs=$rootulibs
```

Comment:
@bellenot do you think we can remove these lines totally?

Reply:
@oshadura probably...